Moderation-Policy.md

```diff
@@ -0,0 +1,80 @@
+## Moderation policy
+
+### Applicability
+
+By default, this policy applies to all repositories under the Node.js Github Organization and all Node.js Working Groups.
+
+Individual Working Groups and Top Level Projects chartered by the TSC may adopt an alternative Moderation Policy for any repository under their stewardship so long as:
+* The Moderation Policy is openly documented as part of the Working Group or Project Charter and;
+* Includes provisions for clearly and openly documenting Moderation actions taken.
+
+### Terms
+
+* "Collaborator" refers to any individual member or "outside collaborator" of the Node.js Github Organization.
+* "Post" refers to the content of any issue, pull request or comment.
+* "Moderate" refers to the act of modifying the content of or deleting any Post.
+* "Ban" refers to the act of blocking an individual Github account from any further participation in the Node.js Github Organization.
+
+### Policy
+
+* All Posts are expected to respect the Node.js Code of Conduct.
+* Only Collaborators with commit rights to a given repository may Moderate Posts within that repository's issue tracker.
+* Only a TSC member may Ban an individual from the Node.js Github Organization.
+* Any individual Banned from the Node.js Github Organization will be recommended for exclusion from any Node.js Foundation sponsored event or activity.
+
+#### Collaborator Posts
+
+* Collaborator's must not Moderate any Post authored by another Collaborator without first giving the author an opportunity of at least 24 hours (from the time of the initial request) to modify or remove the Post on their own.
+* In extreme circumstances the TSC can be consulted to waive the 24 hour grace period.
+* If the author of the Post disagrees that Moderation is required, the matter can be escalated to the TSC by creating a new issue (or using an existing one) within the nodejs/moderation repository with the label `tsc-agenda`. In such cases, the TSC will serve as the final arbiter.
+* For Moderation issues involving a TSC member, the member in question is expected to recuse themselves from any decisions required to resolve the issue.
+* When Moderating any Post authored by another Collaborator, the moderating Collaborator must:
+ * Explain the justification for Moderating the post,
+ * Identify all changes made to the Post, and
+ * Identify the steps previously taken to resolve the issue.
+* Explanations of Moderation actions on Collaborator Posts must be provided in:
+ * A new post within the original thread, or
+ * A new issue within the private nodejs/moderation repository.
+* Any Collaborator who habitually authors Posts that must be Moderated will be subject to possible removal from the Node.js Github Organization. Such action can only be taken through normal TSC motion and vote.
+
```

Comment:
I really like how this list has tightened up. It leaves a lot less to the imagination and clarifies process.

We may want to make a note regarding archiving the original post. That way if there is a disagreement an archive could be used to avoid [gaslighting](https://en.wikipedia.org/wiki/Gaslighting)